tests/functional/sync_v3_settings.js

```diff
@@ -74,7 +74,7 @@ define([
         .then(visibleByQSA('#delete-account .settings-unit-details'))
 
         .then(fillOutDeleteAccount(this, FIRST_PASSWORD))
-        .then(testIsBrowserNotified(this, 'delete_account'))
+        .then(testIsBrowserNotified(this, 'fxaccounts:delete_account'))
```

Comment:
Huh, I just had to do a [search on dxr to make sure I had the story straight](https://dxr.mozilla.org/mozilla-central/search?q=fxaccounts%3Adelete&redirect=false) and in the process realized we have a bug in testing. Through some accident or miscommunication, it came to be that fxaccounts:delete_account is sent for Fennec, fxaccounts:delete for Firefox Desktop. We've had to support both messages since the browsers were released. Since this is a desktop test, we should be checking for fxaccounts:delete, and we need a corresponding fennec test for fxaccounts:delete_account. Adding. Thanks for digging in and making me dive in a bit more deeply.